### ISSUE TITLE
`tidyr 1.3.0` breaks `epipredict`

### ISSUE THREAD

**Issue by brookslogan** [cmu-delphi/epipredict#162]
From tidyr commit fa17c8db357bfb0d0dfe2e962d053395c2c5def3
> * `chop()`, `drop_na()`, `fill()`, `pivot_longer()`, `pivot_wider()`, 
>  `separate_rows()`, `unite()`, `unnest_longer()`, and `unnest_wider()` now
>  error if you supply named selection (#1104).

Leading to:
```
Caused by error in `tidyr::drop_na()`:
! Can't rename variables in this context.
```
running some vignette.

Quick fix is probably pinning tidyr dependency at 1.2.0.  For actual fix, [this post](https://stackoverflow.com/questions/68194439/tidymodels-error-cant-rename-variables-in-this-context) might be relevant.

**Comment by dajmcdon** [comment]
I can't see where this is coming from. `tidyr::drop_na()` is not directly called anywhere in the package. Perhaps in one of the dependencies?

**Comment by brookslogan** [comment]
I believe this was fixed by #165.
